vrchat.jordo.easyquestswitch/Runtime/EQS_Data.cs

```diff
@@ -111,9 +113,27 @@ public void OnChangedBuildTarget(BuildTarget newTarget)
             buildInfo.CachedBuildTarget = buildInfo.NewBuildTarget;
         }
 
+        public void Awake()
+        {
+            if (currentVersion > version)
+            {
+                foreach (Data d in Objects)
+                {
+                    if (d.Type != null)
+                    {
+                        if(d.Type.GetType() == typeof(Type_Material) || d.Type.GetType() == typeof(Type_Material)) 
+                        {
+                            d.Type.Setup(d.Target, version);
+                        }
+                    }
+                }
+                version = currentVersion;
+            }
+        }
+
         public void OnSceneOpened()
         {
-            buildInfo.NewBuildTarget = EditorUserBuildSettings.activeBuildTarget;
+            /*buildInfo.NewBuildTarget = EditorUserBuildSettings.activeBuildTarget;
```

Comment:
Are you keeping this commented-out code to fix and add back in later? If so, I'd prefer to make a Linear task capturing the work to do and moving the WIP code there.

Reply:
It's already existing code for a feature I added in 1.2.0 but that had issues properly working (see https://github.com/vrchat-community/EasyQuestSwitch/issues/13), so for this release I'd like to comment it out (which I had planned to do before but hadn't made a release for it).
I'll make a task.